conf/parser.py

```diff
@@ -107,12 +107,12 @@ def parse(self, ifaces):
 
         # Interface names
         try:
-            self.s1u_ifname = self.conf["s1u"]["ifname"]
-            self.sgi_ifname = self.conf["sgi"]["ifname"]
+            self.access_ifname = self.conf["access"]["ifname"]
+            self.core_ifname = self.conf["core"]["ifname"]
         except KeyError:
-            self.s1u_ifname = "s1u"
-            self.sgi_ifname = "sgi"
-            print('Can\'t parse interface name(s)! Setting it to default values ({}, {})'.format("s1u", "sgi"))
+            self.access_ifname = "access"
+            self.core_ifname = "core"
+            print('Can\'t parse interface name(s)! Setting it to default values ({}, {})'.format("access", "core"))
```

Comment:
I hope this behavior is consistent across the other two controllers as well

Reply:
This should work fine. I tested it with `sim` mode as well. It is working fine. Zmq-based controller should work fine as well. I tested it with Spirent.